model/diagrams/deployment/deply_top-level.plantuml

```diff
@@ -0,0 +1,44 @@
+@startuml
+!include ../exasol.skin
+
+database "ExasolDatabase"
+database "DataSource"  <<external>>
+
+node "Java Virtual Machine" as JVMB {
```

Comment:
This is unclear to me. The VirtualSchemaFrontend (so the database part the adapter talks to) does not run in a JVM. The database (frontend) generates the pushdown request and calls the adapter (backend) with the request as argument. But this frontend is the SQL compiler/optimizer which is written in C++. There is no Java on the database side.

Reply:
Fixed. Note that I also corrected a typo in the file name.